contracts/contracts/ocr2/SequencerUptimeFeed/library.cairo

```diff
@@ -0,0 +1,258 @@
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin
+from starkware.starknet.common.syscalls import (
+    get_tx_info,
+    get_block_timestamp,
+    get_caller_address,
+    get_block_number,
+)
+from starkware.cairo.common.math import assert_not_zero, assert_le
+from starkware.cairo.common.math_cmp import is_le
+from starkware.cairo.common.bool import TRUE, FALSE
+
+from utils import assert_boolean
+from ocr2.interfaces.IAggregator import Round, AnswerUpdated, NewRound
+from SimpleReadAccessController.library import simple_read_access_controller
+from ownable import Ownable_only_owner
+
+@event
+func RoundUpdated(status : felt, transmission_timestamp : felt):
+end
+
+@event
+func UpdateIgnored(
+    latest_status : felt, latest_timestamp : felt, incoming_status : felt, incoming_timestamp : felt
+):
+end
+
+@event
+func L1SenderTransferred(prev : felt, cur : felt):
```

Comment:
There is a subtile API change introduced here: it should be to/from not prev/cur.

Context: https://github.com/smartcontractkit/chainlink/search?q=L1SenderTransferred

Please be careful to keep the original (or close to original) API where appropriate. Please double check the code for those inconsistencies. Thx!

Reply:
`from` is a [system token](https://github.com/starkware-libs/cairo-lang/blob/13cef109cd811474de114925ee61fd5ac84a25eb/src/starkware/cairo/lang/compiler/parser_errors_test.py#L173) and if used not in import(maybe other uses are possible but I've never seen one) results such in error:
`Unexpected token Token('FROM', 'from')`

Reply:
Used @archseer suggestion changed to: `from_addr`, `to_addr`